pages/common/chsh.md

```diff
@@ -1,20 +1,20 @@
 # chsh
 
-> Change the user's login shell.
+> Change a user's login shell.
```

Comment:
How about we drop the article: ~a/the~

Just like we have for `[l]ist available shells` :)
```suggestion
> Change user's login shell.
```